.github/workflows/main.yml

```diff
@@ -22,8 +23,8 @@ jobs:
         uses: actions/checkout@v4
 
       - name: generate SBOM
-        uses: manifest-cyber/manifest-github-action@main
+        uses: manifest-cyber/manifest-github-action@fix-trivy-default
```

Comment:
Can this be reverted before the actual merge, or do you want to do a separate PR for that?